app/src/main/AndroidManifest.xml

```diff
@@ -5,9 +5,10 @@
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
-    <uses-feature android:name="android.hardware.camera.any" />
-
     <uses-permission android:name="android.permission.CAMERA" />
+    <uses-permission android:name="android.permission.INTERNET" />
+
```

Comment:
Why do you need the Internet permission? Is it useful for this PR?

Reply:
It was written in the developer's guide, I believe it is necessary to access your Facebook account

Reply:
Alright, that makes sense - thanks!